Cargo.toml

```
@@ -13,7 +13,7 @@ categories.workspace = true
 readme.workspace = true
 
 [workspace.package]
-version = "0.16.0"
+version = "0.16.1"
```

FYI we have an integration with release-plz, which will automatically handle all of the version bumping, for example the workspace crates should have all the same version numbers and once this merges this will be done automatically :)

```suggestion
version = "0.16.0"
```

---

nice, rolled back.